stock_move_location/wizard/stock_move_location.py

```diff
@@ -199,7 +234,8 @@ def _get_group_quants(self):
         # Using sql as search_group doesn't support aggregation functions
```

Comment:
I know it is not a modification of this PR, but I don't quite understand what cannot be done with read_group

Reply:
Indeed, you're right. Something to be done in another PR ? As this is bypassing security.